sections/semantics-document-metadata.include

```diff
@@ -823,9 +823,11 @@
   
   <div class="warning">
   
-  <p>Developers should not use the <code>user-scalable=no</code> or <code>maximum-scale=1.0</code>
```

Comment:
This deletion removes the normative "should not" needs to be worked into updated prose.